AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -20,4 +22,25 @@
  */
 - (void)recursivelyEnsureDisplaySynchronously:(BOOL)synchronously;
 
+- (void)drawRect:(CGRect)bounds withParameters:(id <NSObject>)parameters isCancelled:(asdisplaynode_iscancelled_block_t)isCancelledBlock isRasterizing:(BOOL)isRasterizing;
```

Comment:
These method signatures look great, thank you for adding them! We might want to put the stuff in framework private for now, although I suppose it is reasonable to put it on a track to eventually make public. Mainly thinking that we won't really recommended for use, Because it should be quite rare to have a custom drawing node that occurs as frequently as image and text nodes that have demonstrated a performance hotspot in creating the display parameters on the main thread.